core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -406,7 +406,7 @@ public void listContextJsps(Context context, Summary summary, boolean compile) {
   }
 
   @Override
-  public org.apache.juli.logging.Log getLogger(Context context) {
+  public Object getLogger(Context context) {
```

Comment:
Because it is defined that way on TomcatContainer interface and every usage of that method, 3 in total, uses object either. I tried not to change the system "default" usage. We could address this in another issue maybe.